src/heretic/model.py

```diff
@@ -39,7 +39,8 @@ def __init__(self, settings: Settings):
         print(f"Loading model [bold]{settings.model}[/]...")
 
         self.tokenizer: PreTrainedTokenizerBase = AutoTokenizer.from_pretrained(
-            settings.model
+            settings.model,
+            trust_remote_code=settings.trust_remote_code,
```

Comment:
:white_check_mark: Completely safe so far, will simply follow the `trust_remote_code` setting, defaulting to `None`, which prompts the user.